shared/reducers/chat2.tsx

```diff
@@ -616,6 +579,7 @@ const reducer = Container.makeReducer<Actions, Types.State>(initialState, {
       flipStatusMap.set(status.gameID, status)
     })
   },
+  /// AAAA here
```

Comment:
?

Reply:
Sorry that was me tracking where I was when switching contexts

Reply:
🔥 :)